# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/palo/default-to-boot.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

As I nearly never need '0' to edit the kernel, let's default to a
IMHO much more reasonable 'b' to boot.

--- palo-2.24/ipl/ipl.c.vanilla	2024-04-06 13:50:21.630969647 +0200
+++ palo-2.24/ipl/ipl.c	2024-04-06 13:51:58.863969220 +0200
@@ -406,7 +406,7 @@
 	    "'d'           toggle debug mode\n"
 #endif
 	    "? ");
-	numbuf[0] = '0';
+	numbuf[0] = 'b';
 	numbuf[1] = '\0';
 	enter_text(numbuf, sizeof numbuf - 1);
 	printf("\n");
